lib/pipelines/complexity.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var path = require('path');
+var plato = require('es6-plato');
+var through = require('through2');
+
+module.exports.plato = function(options) {
+	var name;
+
+	try {
+		name = require(path.join(process.cwd(), 'package.json')).name;
+	}
+	catch (e) {}
+
+	var files = [];
+	var platoArgs = {
+		complexity: {},
+		title: name || 'Metal'
+	};
+
+	return through.obj(function(file, encoding, callback) {
+		files.push(file.path);
+		callback();
+	}, function(callback) {
+		var stream = this;
+
+		plato.inspect(files, options.dest, platoArgs, function() {
```

Comment:
Since this already takes both a src and a dest, and we don't do any other transformations in the middle, seems like we don't even need to create this `through.obj` pipeline. The `README.md` file from [`es6-plato`](https://www.npmjs.com/package/es6-plato) suggests using it like this with gulp:

```js
gulp.task(taskPrefix + 'complexity:report', function() {
  return plato.inspect(options.complexitySrc, options.complexityDest, platoArgs);
});
```

That way we can even get rid of this file, and the final code will be simpler :)